static/test/javascripts-legacy/spec/common/onward/tech-feedback.spec.js

```diff
@@ -1,83 +1,62 @@
         var TechFeedback;
 
+        var fixturesConfig = {
+            id: 'related',
+            fixtures: [
+                '<p id="feedback-warning"></p>',
+                '<select id="feedback-category"></select>',
+                '<div id="feedback-form-default"></div>'
+            ]
+        };
+
         beforeEach(function(done) {
            injector.mock('svgs/icon/thumb.svg', {
                markup: ''
            });
            injector.require(['common/modules/onward/tech-feedback'], function(TechFeedbackModule) {
                TechFeedback = TechFeedbackModule;
+               fixtures.render(fixturesConfig);
                done();
            })
         });
 
-        it('should exist', function () {
-            expect(TechFeedback).toBeDefined();
+        afterEach(function () {
+            fixtures.clean(fixturesConfig.id);
         });
 
-        it('should be able to get no params from the empty hash', function () {
-
-            var t = new TechFeedback();
-            var values = t.getValuesFromHash('');
-            expect(values).toEqual({});
-        });
-
-        it('should be able to get one param from the hash', function () {
-
-            var t = new TechFeedback();
-            var values = t.getValuesFromHash('#page=cheese');
-            expect(values).toEqual({'page': 'cheese'});
+        it('Should exist', function () {
+            expect(TechFeedback).toBeDefined();
         });
 
-        it('should be able to get more than one param from the hash', function () {
-
-            var t = new TechFeedback();
-            var values = t.getValuesFromHash('#page=cheese&url=milk');
-            expect(values).toEqual({'page': 'cheese', 'url': 'milk'});
+        it("Should return a browser in extra information", function(){
+            var feedback = new TechFeedback();
+            expect(feedback.getExtraDataInformation().browser).toBeDefined();
         });
 
-        it('should decide uri components', function () {
-
-            var t = new TechFeedback();
-            var values = t.getValuesFromHash('#http%3A%2F%2Fwww.theguardian.com=http%3A%2F%2Fwww.theguardian.com');
-            expect(values).toEqual({'http://www.theguardian.com': 'http://www.theguardian.com'});
+        it("Should recognise a lack of AB tests", function(){
+            var feedback = new TechFeedback();
+            expect(feedback.getExtraDataInformation().abTests).toBe("No tests running");
         });
 
-        it('should be able to get ads creative IDs', function () {
-
-            var t = new TechFeedback();
-            var values = t.getValuesFromHash('#ads=67729052367%2C71293284447');
-            expect(values).toEqual({'ads': '67729052367,71293284447'});
+        it("Should be able to summarise AB tests if the exist", function(){
+            var feedback = new TechFeedback();
+            expect(feedback.summariseAbTests({Foo : {variant : 'foo'}, Bar : {variant : 'bar'}})).toBe('Foo=foo, Bar=bar');
```

Comment:
We can test this without exposing `summariseAbTests`:

1. Mock out the `common/modules/experiments/ab` module
2. Provide a `getParticipations` method on the mock that returns the test data above
3. Test that the expected stringified data has been set as the value for `.feedback__form input[name=extra]`

Let me know if you'd like to pair on this